plugins/connection/aws_ssm.py

```diff
@@ -360,10 +355,12 @@
 
 from ansible_collections.amazon.aws.plugins.module_utils.botocore import HAS_BOTO3
 
+from ansible_collections.community.aws.plugins.module_utils.s3clientmanager import S3ClientManager
```

Comment:
what if you move this to `plugin_utils` like for `amazon.aws` collection?

Reply:
imo for now module_utils seem a reasonable place, as we will be adding more classes like `SSMSessionManager ` and `FileTransferManager ` with other jira stories.
But I'm open to either module_utils or creating new dir plugin_utils.
@abikouo @alinabuzachis

Reply:
If I remember correctly, I had to put the amazon.aws pieces into plugin_utils rather than module_utils, because the linter will raise an error if you import from something that **won't** be sent to a remote host.

Personally I would recommend moving this into plugin_utils.

Over in amazon.aws.plugins.plugin_utils we also have the initial framework for an "AWS" connection plugin (plugin_utils.connection.AWSConnectionBase), which _might_ simplify some of this code further.

Reply:
moved to `plugin_utils`.